manifests/prombench/stateful-set.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: prombench-test-{{ .PR_NUMBER }}
   labels:
     app: prombench-test-{{ .PR_NUMBER }}
+  annotations:
+    prometheus.io/prombench.exit_on_error: 'true'
```

Comment:
don't we want this in all cases?

Reply:
I thought so too, should the default behavior be changed then?

The only time I found NOT exiting on error useful was the time `/manifest/prombench/benchmark` is getting deployed and if some deployment times out, the other deployment after that continue to get deployed. (but this behavior is undesirable sometimes) let me know if we should change this.

If we change this then we also won't need the global var that you mentioned : `ExitOnError = false `